docs/README.md

```diff
@@ -104,63 +104,110 @@ We will review your changes, add any feedback and once everything looks good mer
 
 ## Advanced Usage
 
+_Note: YCQL docs are still using an old method.  Follow these instructions for YSQL, but use these
+instructions just as a reference for YCQL._
+
 ### Generate API syntax diagrams
 
-To update the documentation with new grammars and syntax diagrams, follow these steps:
+1. Download the latest RRDiagram JAR file (`rrdiagram.jar`).  You can find it on the [release
+   page](https://github.com/YugaByte/RRDiagram/releases/), or you can try running the following
+   command.
 
-_Note: Modifications will typically be added to the `ysql` or `ycql` subdirectories of the `docs/content/latest/api/` directory._
+   ```sh
+   wget $(curl -s https://api.github.com/repos/YugaByte/RRDiagram/releases/latest \
+          | grep browser_download_url | cut -d \" -f 4)
+   ```
 
-1. Update the appropriate EBNF source grammar file (`<source>_grammar.ebnf`) with your changes, for example, adding support for a new statement or clause.
+   _Note: Alternatively, you can manually build the jar file as described in the [build
+   section](https://github.com/YugaByte/RRDiagram/README.md#build) of the `RRDiagram` repo (and
+   move/rename the resulting jar from the target folder)._
 
-    - YSQL API: `./content/latest/api/ysql/syntax_resources/ysql_grammar.ebnf`
-    - YCQL API: `./content/latest/api/ycql/syntax_resources/ycql_grammar.ebnf`
+1. Run the diagram generator using the following command:
```

Comment:
Should this be `2.`? -- Same below, looks like all numbers are `1.`

Reply:
That numbering is valid.  Take a look at the [file] to see the "2.".  I prefer it because it easily allows insertions into or deletions from the list.

[file]: https://github.com/jj-kim/yugabyte-db/blob/docs-readme-reformat/docs/README.md#generate-api-syntax-diagrams